app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -3,6 +3,7 @@ package org.simple.clinic.drugs
 import androidx.test.runner.AndroidJUnit4
```

Comment:
`Provide update prescription` doesn't… make sense. Did you miss some words in the middle?